src/main/java/ee/developest/gtm/popup/PopupFactory.kt

```diff
@@ -11,12 +11,25 @@ class PopupFactory {
     companion object {
         private const val GROUP_ID = "gtm"
 
-        fun showInitConfirmation(project: Project, onYes: () -> Unit, onNo: () -> Unit?) {
-            val res = Messages.showDialog(project, "Do you want to initialize gtm time tracking for current repository?", "Gtm Enhanced", arrayOf("Yes", "No"), 0, Messages.getQuestionIcon())
-            if (res == 0) {
-                onYes()
-            } else {
-                onNo()
+        fun showInitConfirmation(project: Project, onYes: (local: Boolean) -> Unit, onNo: () -> Unit?) {
+            val res = Messages.showDialog(
+                project,
+                "Do you want to initialize gtm time tracking for current repository?",
+                "Gtm Enhanced",
+                arrayOf("Locally", "Globally", "No"),
```

Comment:
I'd change to ` "Yes", "Only locally", "No"`.
It's kinda debatable tho, so if you feel differently, comment below

Reply:
I figured that most users probably want to initialize gtm locally, and figured that with the "yes" option alone, the user might not necessarily know that the data is being pushed.

Reply:
Ok, let it be then.
I need to think how to encourage peale to use also our web UI for public projects as well then :p